<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load_image">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>加载图片</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="image_path">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>551</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="canny_bt">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Canny算法</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hough_bt">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Hough变换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="execute_bt">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>一键处理</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>290</y>
      <width>67</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>预测结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gaussian_filiter_bt">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>高斯滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sobel_filiter_bt">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>计算梯度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="non_max_suppression_bt">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>非极大化</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threshold_bt">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>双阈值法</string>
    </property>
   </widget>
   <widget class="QPushButton" name="soft_vote_bt">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>软投票法</string>
    </property>
   </widget>
   <widget class="QPushButton" name="non_max_suppression_bt2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>210</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>非极大化</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="resut_text">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>371</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enlarge_bt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>放大查看-结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setParam_bt">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>设置参数</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
